packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilter.ts

```diff
@@ -520,7 +527,11 @@ function toGenericInstanceFilterRuleValue(primitiveValue?: PrimitiveValue): Gene
     return undefined;
   }
 
-  return { displayValue: primitiveValue.displayValue ?? "", rawValue: primitiveValue.value };
+  return {
+    displayValue: primitiveValue.displayValue ?? "",
+    rawValue: primitiveValue.value,
+    ...(primitiveValue.roundingError ? { roundingError: primitiveValue.roundingError } : undefined),
```

Comment:
`roundingError` is numeric, so I think it's better to check if it exists rather for truthiness

```suggestion
    ...("roundingError" in primitiveValue ? { roundingError: primitiveValue.roundingError } : undefined),
```